full_dependencies.conf.json

```diff
@@ -0,0 +1,169 @@
+{
+    // VALUES WE CURRENTLY SPECIFY
+
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "PennyLane",
+
+    // The project's homepage
+    "project_url": "pennylane.ai",
+
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    "repo": "https://github.com/PennyLaneAI/pennylane.git",
+
+    // The directory (relative to the current directory) that benchmarks are
+    // stored in.  If not provided, defaults to "benchmarks"
+    "benchmark_dir": "benchmarks",
```

Comment:
Do we need to change this? Looks like the default is this (https://asv.readthedocs.io/en/stable/asv.conf.json.html#benchmark-dir)